packages/by-name/contrast/package.nix

```diff
@@ -73,11 +73,28 @@ buildGoModule rec {
     install -D ${genpolicy.rules}/genpolicy-rules.rego cli/cmd/assets/genpolicy-rules.rego
   '';
 
+  # get the replacements and corresponding container versions
+  imageReplacementsList = lib.splitString "\n" (builtins.readFile ../../../cli/cmd/assets/image-replacements.txt);
+  imageVersionsList = builtins.map (x: lib.last (lib.splitString "=" x)) imageReplacementsList;
+
+  # build a string to embed into the CLI. Since embedding does not allow newlines it is split again in the go program.
+  imageVersionsCommas = lib.removeSuffix "," (builtins.concatStringsSep "," imageVersionsList);
+
+  # `nix build .#contrast` should also work if image-replacements.txt does not contain release info
+  imageVersionsString =
+    if builtins.head (lib.stringToCharacters imageVersionsCommas) == "#"
+    then "will-be-replaced-in-release@sha256:0000000000000000000000000000000000000000000000000000000000000000"
+    else imageVersionsCommas;
+
   CGO_ENABLED = 0;
   ldflags = [
     "-s"
     "-w"
     "-X main.version=v${version}"
+    "-X main.runtimeHandler=${runtimeHandler}"
+    "-X main.launchDigest=${launchDigest}"
+    "-X main.genpolicyVersion=${genpolicy.version}"
+    "-X main.containerVersions=${imageVersionsString}"
```

Comment:
The image replacements are already embedded in the CLI in the `cmd` package. We can just make it public and use that.